### PR TITLE
 **JavaScript: add JSON.stringify article**

### PR DESCRIPTION
**Pull request by Parama92**
Added a guide article for `JSON.stringify()`

---

## ✅️ By submitting this PR, I have verified the following

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md` since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Other examples are **Git: edit Git Commit article** or **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate or repetitive  content that has been directly copied from another source.

### CODE REVIEW

File: src/pages/javascript/standard-objects/json/json-stringify/index.md [hunk 1]
```diff
@@ -3,13 +3,189 @@ title: JSON Stringify
...
+- `function(){}`
+- (ES6+) `Symbol`
+- An object with circular reference(s) in it
+
+#### Syntax
+
+```javascript
+  JSON.stringify( value [, replacer [, space]])
+```
+In it's simplest and most used form:
```
Comment (amandeepmittal) [inline src/pages/javascript/standard-objects/json/json-stringify/index.md]: Just a little grammatical change: _In it's ..._ could be **In its ...**

**Comment by Parama92** [comment]
Oops! My bad. Changing right away!

Regards.

**Comment by amandeepmittal** [comment]
Great work 👍

**Comment by Parama92** [comment]
Thanks for the feedback!

**Comment by amandeepmittal** [comment]
@nsuchy I do not have rights to merge PR in this repository. That is why I could not do it. I mostly maintain #100DaysOfCode repo. Can I get access? I'd love to help 😄

**Comment by nsuchy** [comment]
@amandeepmittal You would probably need to talk to @Bouncey about that, he's in charge of this project. I'll go ahead and merge this PR though. Thanks for the support 👍

**Comment by amandeepmittal** [comment]
@nsuchy thanks for the tip!

**Pull request merged**